scripts/README.md

```diff
@@ -0,0 +1,52 @@
+# Setup & Build scripts
+
+This directory contains a few scripts to simplify setting up the Windows agent
+of webpagetest.
+
+The scripts have been developed and tested on Windows 8.1 Enterprise
```

Comment:
We will need to test on a Windows 2012 and Windows 2012 R2 server. I can provide you with spun up instances if needed.